src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
The class is a little bare bone :-). Consider adding validation, toString() methods.